MODULE.bazel

```diff
@@ -2,8 +2,17 @@
 
 module(
     name = "bazel",
-    version = "7.0.0-pre",
     repo_name = "io_bazel",
+    version = "7.0.0-pre",
+)
+
+archive_override(
+    integrity = "sha256-L9ASXKv/pEv9VKLmueHxobnl5aCWk4MNTZywE0MKIe4=",
+    module_name = "rules_cc",
+    strip_prefix = "rules_cc-ffaaf6bfb168eef354ae3b089085ccb86b47d1da",
+    urls = [
+        "https://github.com/keith/rules_cc/archive/ffaaf6bfb168eef354ae3b089085ccb86b47d1da.tar.gz",
```

Comment:
Presumably this needs to be a released rules_cc at some point, just flagging so it's not forgotten.

Reply:
yep! Hoping that @oquenchil can help with a release after https://github.com/bazelbuild/rules_cc/pull/154 is merged (which hopefully is in progress)